packages/router/src/RouterModule.ts

```diff
@@ -56,18 +60,35 @@ export class RouterModule extends YabModule<RouterConfig> {
 			middlewares: options?.middlewares,
 			errorHandler: options?.errorHandler,
 			responseHandler: options?.responseHandler,
+			customValidation: async (schema: TSchema, value: unknown) => {
```

Comment:
this value is: `options?.customValidation` as use created. it can't not override

the function we need is another function, that is a composition of `options?.customValidation || validatePayloadRequest`